torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1001,6 +1001,16 @@ def sample_inputs_pinverse(op_info, device, dtype, requires_grad=False):
                        SkipInfo('TestUnaryUfuncs', 'test_reference_numerics',
                                 dtypes=[torch.bfloat16]),
                    )),
+    UnaryUfuncInfo('rsqrt',
+                   ref=lambda x: np.reciprocal(np.sqrt(x)),
+                   domain=(1e-15, float('inf')),
```

Comment:
Since 1e-15 isn't representable in float16 isn't this effectively a domain of (0, inf)?

Reply:
Also, seems like the domain should be (0, inf), because rsqrt(0) = inf, not NaN

Reply:
Right. Same the ASAN was failing with division by zero. Plus `0` is not actually in domain of `rsqrt`. So thought of having something slightly above zero. 

Will verify once after turning `0, inf` that everything is fine.

Reply:
It's true that it's not in the mathematical domain but I think our "domain" is just wherever the function doesn't map to NaN.